docs/source/ryzenai/usage_guides/pipelines.mdx

```diff
@@ -0,0 +1,123 @@
+# Ryzen Inference pipelines
+
+The pipelines are a great and easy way to use models for inference. These pipelines are objects that abstract most of the complex code from the library, offering a simple API dedicated to several tasks.
+
+Currently the supported tasks are:
+
+* `image-classification`
+* `object-detection` (Supported model types - yolox, yolov3, yolov5, yolov8)
+
+<Tip warning={true}>
+The RyzenAI pipelines requires a runtime configuration file for inference. A default version of this runtime configuration file can be found in the Ryzen AI VOE package, extracted during installation under the name `vaip_config.json`.
+
+For more information refer to [runtime-configuration-file](https://ryzenai.docs.amd.com/en/latest/runtime_setup.html#runtime-configuration-file)
+</Tip>
+
+## The pipeline abstraction
+The pipeline abstraction is a wrapper around all the available pipelines for specific tasks. The pipeline() function automatically loads a default model and tokenizer/feature-extractor capable of performing inference for your task.
```

Comment:
You could link to Transformers documentation about pipelines, or mention that this is derived from it